fast64_internal/f3d/f3d_material.py

```diff
@@ -357,6 +357,7 @@ def ui_geo_mode(settings, dataHolder, layout, useDropdown):
 		inputGroup.prop(settings, 'g_tex_gen_linear',
 			text = 'Texture UV Generate Linear')
 		inputGroup.prop(settings, 'g_shade_smooth', text = 'Smooth Shading')
+		inputGroup.prop(settings, 'g_celshading', text = 'Cel Shading (Shade G -> A)')
```

Comment:
i think we ought to have all cel shading UI hidden from a global setting in the Fast64 menu:

![image](https://user-images.githubusercontent.com/79979276/178126811-d3daf6e8-3270-481f-9f74-55edf5c980de.png)

Either a new "microcode", or a checkbox. otherwise beginners or people unaware may check this option and be confused.

Reply:
Fixed